platform/security/handler/security-handler-api/src/main/java/org/codice/ddf/security/handler/api/HandlerResult.java

```diff
@@ -67,7 +69,7 @@ public String toString() {
     sb.append("; Source: ");
     sb.append(source);
     sb.append("; Token: ");
-    sb.append(token.toString());
+    sb.append(Objects.isNull(token) ? "null" : token.toString());
```

Comment:
A null token prevented the object from being displayed in the IDE watches/variables view.